packages/primitives/src/Menu/MenuItem.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+import { SystemProps, VariantProps } from '@kodiak-ui/core'
+import { Box } from '../Box'
+import { Link } from '../Link'
+
+type MenuItemProps = {
+  children: React.ReactNode
+  isCurrent?: boolean
+  dismissLabel?: string
+  onDismiss?: () => void
+  as?: React.ElementType
+  href?: string
+} & React.HTMLAttributes<HTMLDivElement> &
+  VariantProps &
+  SystemProps
+
+export const MenuItem = React.forwardRef<HTMLDivElement, MenuItemProps>(
+  function Message(
+    {
+      children,
+      isCurrent,
+      variantKey = 'menuitem',
+      variant,
+      as: renderAs = 'a',
+      href,
+      ...props
+    },
+    forwardedRef,
+  ) {
+    return (
+      <Box
+        variantKey={variantKey}
+        variant={variant}
+        ref={forwardedRef}
+        {...props}
+      >
+        <li>
+          {renderAs === 'a' && (
```

Comment:
@wesleycole I'm tempted to just take children and render them but then it's up to the calling component to set up some link defaults as well as the aria-current.  It then becomes more important to look at the storybook/docs to see the sample of how it should be rendered.

Reply:
Yep, as we discussed in Slack, let's just accept `children` for now and its up to the developer to render the inside.